### PR TITLE
Add modules for Raspberry Pi 4

### PR DESCRIPTION
**Pull request by gajavrbinc**
When builing with bazel, the target which is defined in the executed command, is searched in BUILD.bazel, which has specified all of the needed output files and points to build.config.rpi4 which points to defconfig file to compile all of the drivers and modules. 
Based on v5.15.94 branch's bcm2711_aosp14_defconfig configuration file, all od the modules were added to android_rpi4_defconfig as "=m". When building with bazel, these modules get compiled and give message which modules and their paths need to be added to BUILD.bazel. Besides that, goal to build modules needed to be added to BUILD.bazel as well as to build.config.rpi4.

### CODE REVIEW

File: BUILD.bazel [hunk 1]
```diff
@@ -1050,9 +1196,12 @@ kernel_build(
     ],
     build_config = "build.config.rpi4",
     make_goals = [
+         "modules",
          "Image.gz",
          "dtbs",
     ],
+    module_outs = _RPI4_MODULE_OUTS,
+    strip_modules = True,
```
Comment (JanKehren) [inline BUILD.bazel:1204]: @gajavrbinc what does this do?

File: BUILD.bazel [hunk 2]
```diff
@@ -1027,6 +1027,152 @@ _RPI_OVERLAY_OUTS = [
...
+    "drivers/rtc/rtc-max6900.ko",
+    "drivers/pwm/pwm-pca9685.ko",
+    "drivers/rtc/rtc-ds1374.ko",
+    "sound/soc/codecs/snd-soc-pcm186x.ko",
+    "sound/soc/generic/snd-soc-simple-card.ko",
+    "sound/soc/codecs/snd-soc-pcm512x-i2c.ko",
+    "sound/soc/codecs/snd-soc-pcm512x.ko",
+    "sound/soc/codecs/snd-soc-pcm179x-codec.ko",
+    "lib/crypto/libcurve25519-generic.ko",
+]
```
Comment (JanKehren) [inline BUILD.bazel:1174]: @gajavrbinc how did you create this list?

File: arch/arm64/configs/android_rpi4_defconfig
Comment (JanKehren) [inline arch/arm64/configs/android_rpi4_defconfig:1]: @gajavrbinc can we rename this file to our old naming scheme?

File: BUILD.bazel (hunk 1 above)
Comment (gajavrbinc) [reply BUILD.bazel:1204]: @JanKehren This line basically stripps away unnecessary symbols. It decreases size. False is mostly only used for debbuging.

File: BUILD.bazel (hunk 2 above)
Comment (gajavrbinc) [reply BUILD.bazel:1174]: @JanKehren When compiling using the config file, bazel finds the modules (=m), returns an error and writes these paths based on which modules were specified in the config file. After adding these correct paths, bazel compiled succesfully.

File: arch/arm64/configs/android_rpi4_defconfig
Comment (gajavrbinc) [reply arch/arm64/configs/android_rpi4_defconfig:1]: @JanKehren sure, I renamed it

**Pull request merged**